bitnami/suitecrm/templates/pv.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if and .Values.persistence.enabled .Values.persistence.hostPath (not .Values.persistence.existingClaim) -}}
```

Comment:
We usually don't create the PV but the PVC

Reply:
It is used for supporting `hostPath`